src/interface.jl

```diff
@@ -81,3 +81,73 @@ to checking if the value returned by `param_sym_to_index` is not `nothing`.
 function is_param_sym end
 
 is_param_sym(store, sym) = !isnothing(param_sym_to_index(store, sym))
+
+"""
+$(TYPEDSIGNATURES)
+
+Get an iterable over the observed variable expressions for the given system.
+Default to an empty vector.
+"""
+function observed end
+
+observed(::Any) = []
```

Comment:
Why not empty tuple?

Reply:
Just following the pattern for the other defaults, I can change it if tuple is better?